pkg/audit/service.go

```diff
@@ -5,24 +5,45 @@ package audit
 
 import (
 	"context"
+	"time"
+
+	"go.uber.org/zap"
 
 	"storj.io/storj/pkg/overlay"
 	"storj.io/storj/pkg/pointerdb/pdbclient"
 	"storj.io/storj/pkg/provider"
 	"storj.io/storj/pkg/transport"
+	"storj.io/storj/pkg/utils"
 )
 
 // Service helps coordinate Cursor and Verifier to run the audit process continuously
 type Service struct {
 	Cursor   *Cursor
 	Verifier *Verifier
 	Reporter reporter
+	ctx      context.Context
```

Comment:
curious why do you have the context on the struct ?

Reply:
I wanted access to the `cancel()` is there a better way to structure the `Run` function?

Reply:
can't you just pass it ?

Reply:
I think the issue is it's unclear which context this refers to and allows it to be over written accidentally by someone coming in later not knowing how downstream processes are using it. Looks like you need to know when it's cancelled in the go func in the Run function, so just pass it directly there.